dashboard/app/models/teacher_feedback.rb

```diff
@@ -34,6 +34,13 @@ class TeacherFeedback < ApplicationRecord
   belongs_to :level
   belongs_to :teacher, class_name: 'User'
 
+  REVIEW_STATES = [
+    "KeepWorking",
```

Comment:
Mostly curious, why did you choose PascalCase here?

Reply:
Good question. I was thinking the performance_level was PascalCase but it's actually camelCase. For consistency I should probably use that. I'll update.

Reply:
Thanks for pointing this out!